OpenRA.Game/Player.cs

```diff
@@ -113,6 +113,18 @@ static FactionInfo ChooseDisplayFaction(World world, string factionName)
 			return factions.FirstOrDefault(f => f.InternalName == factionName) ?? factions.First();
 		}
 
+		static string ChooseBot(World world, string botType)
+		{
+			if (botType == null || botType != "random")
```

Comment:
```suggestion
			if (botType != "random")
```

as `null` is also `!= "random"`.